src/app/explore/explore.component.ts

```diff
@@ -37,47 +77,177 @@ export class ExploreComponent implements OnDestroy, OnInit {
+   */
+  protected readonly campaignIdsWithRectangleImage: string[] = environment.environmentId !== 'production' ?
+    [
+      'a056900002RXrXtAAL',
+      'a056900002SEVVPAA5', // Christmas Challenge 2024
+    ] :
+    [
+      'a056900002SEVVPAA5', // Christmas Challenge 2024
+    ];
+
   constructor(
     private campaignService: CampaignService,
+    private currencyPipe: CurrencyPipe,
     private datePipe: DatePipe,
+    private fundService: FundService,
+    private navigationService: NavigationService,
     private route: ActivatedRoute,
     private router: Router,
     private pageMeta: PageMetaService,
     private scroller: ViewportScroller,
     public searchService: SearchService,
     @Inject(PLATFORM_ID) private platformId: Object,
+    private state: TransferState,
+    @Inject(openPipeToken) private timeLeftToOpenPipe: TimeLeftPipe,
+    @Inject(endPipeToken) private timeLeftToEndPipe: TimeLeftPipe,
+    @Inject(SESSION_STORAGE) private sessionStorage: StorageService,
   ) {}
 
   ngOnDestroy() {
+    if (isPlatformBrowser(this.platformId) && this.tickerUpdateTimer) {
+      clearTimeout(this.tickerUpdateTimer);
+    }
+
     this.routeParamSubscription?.unsubscribe();
     this.searchServiceSubscription?.unsubscribe();
     this.queryParamsSubscription?.unsubscribe();
+    this.routeChangeListener?.unsubscribe();
+  }
+
+  protected get title() {
+    if (! this.metaCampaign) {
+      return 'Big Give';
+    }
+
+    if (! this.fund) {
+      return this.metaCampaign?.title;
+    }
+
+    // Show fund name if applicable *and* there's no fund logo. If there's a logo
+    // its content + alt text should do the equivalent job.
+    return (!this.fund.logoUri && this.fund.name)
+      ? `${this.metaCampaign.title}: ${this.fund.name}`
+      : this.metaCampaign.title;
   }
 
   ngOnInit() {
-    this.pageMeta.setCommon(
-      'Big Give',
-      'Big Give – discover campaigns and donate',
-      'https://images-production.thebiggive.org.uk/0011r00002IMRknAAH/CCampaign%20Banner/db3faeb1-d20d-4747-bb80-1ae9286336a3.jpg',
-    );
+    this.metaCampaign = this.route.snapshot.data.campaign;
 
-    this.loadQueryParamsAndRun();
+    this.listenForRouteChanges();
+
+    this.setSecondaryPropsAndRun(this.metaCampaign);
+
+    let fundKey: StateKey<Fund>;
+    if (this.fundSlug && this.metaCampaign) {
+      fundKey = makeStateKey<Fund>(`fund-${this.fundSlug}`);
+      this.fund = this.state.get<Fund | undefined>(fundKey, undefined);
+      if (this.fund) {
+        this.setFundSpecificProps(this.fund, this.metaCampaign);
+      }
+    }
 
     this.beneficiaryOptions = CampaignGroupsService.getBeneficiaryNames();
     this.categoryOptions = CampaignGroupsService.getCategoryNames();
     this.locationOptions = CampaignGroupsService.getCountries();
-
     this.queryParamsSubscription = this.scrollToSearchWhenParamsChange();
-    this.highlightCards = this.route.snapshot.data.highlights;
```

Comment:
I think we've maybe accidentally lost assigning the highlight cards as a fallback in Explore mode? Seems like the router still has the resolver but we need to assign it here.

Reply:
Thanks, fixed in pairing with @dorota-joanna yesterday.